proposals/json-td/index.html

```diff
@@ -71,37 +79,59 @@
       "type": "boolean,
       "description": "Whether the lamp is turned on",
       "href": "/things/lamp/properties/on"
+    },
+    "lightLevel" :
+    {
+      "type": "integer",
+      "description": "The level of light from 0-100",
+      "min" : 0,
+      "max" : 100,
+      "href": "/things/lamp/properties/lightLevel"
     }
   },
   "actions": {
     "toggle": {
+      "type" : "boolean",
       "description": "Toggle the lamp on and off"
+    },
+    "dim": {
+        "type" : "integer",
+        "min" : 0,
+        "max" : 100,
+        "description": "Change the level of light"
     }
   },
   "events": {
     "overheating": {
       "description": "The lamp has exceeded its safe operating temperature"
     }
+  },
+  "links": {
+    "properties": "/thing/pi/properties",
+    "actions": "/things/pi/actions",
+    "events": "/things/pi/events"
   }
 }
+</section>
         </pre>
       </div>
 
       <section>
         <h3><code>name</code> member</h3>
-        <p>The name member is a user friendly string which describes the device. This can be set to any value by the device creator and may include a brand name or model number.</p>
+        <p>The name member is a user friendly string which identifies the device. This can be set to any value by the device creator and may include a brand name or model number.</p>
```

Comment:
Makes sense. Maybe we should change this to "human readable" rather than "user friendly" as in the Web Thing Model.